packages/twenty-front/src/modules/ui/field/display/components/JsonDisplay.tsx

```diff
@@ -0,0 +1,14 @@
+import { OverflowingTextWithTooltip } from '../../../display/tooltip/OverflowingTextWithTooltip';
+
+import { EllipsisDisplay } from './EllipsisDisplay';
+
+type JsonDisplayProps = {
+  text: string;
+  maxWidth?: number;
+};
+
+export const JsonDisplay = ({ text, maxWidth }: JsonDisplayProps) => (
+  <EllipsisDisplay maxWidth={maxWidth}>
+    <OverflowingTextWithTooltip text={text} mutliline />
```

Comment:
I don't think we need the tooltip as it's computed on width only and not on height so this implementation doesn't make that much sense. You can keep the changes on {children} though

But replace this by just `  <EllipsisDisplay maxWidth={maxWidth}>{text}</EllipsisDisplay>`

Reply:
On this